spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1606,6 +1631,20 @@ private Table createTypeTestTable() {
     return table;
   }
 
+  private void insertData(int recordCount, int numDataFiles) {
```

Comment:
Why do we need this function? Doesn't "Write Records" do this for us?

Reply:
dropped this function as we dont need this